src/test/java/org/gridsuite/modification/server/modifications/LineSplitWithNewVoltageLevelTest.java

```diff
@@ -53,20 +55,27 @@ protected Network createNetwork(UUID networkUuid) {
     @Override
     protected ModificationInfos buildModification() {
         VoltageLevelCreationInfos vl1 = VoltageLevelCreationInfos.builder()
-            .equipmentId("vl1")
-            .equipmentName("NewVoltageLevel")
-            .nominalVoltage(379.3)
-            .substationId("s1")
-            .busbarSections(List.of(new BusbarSectionCreationInfos("v1bbs", "BBS1", 1, 1)))
-            .busbarConnections(List.of())
-            .build();
+                .equipmentId("newVoltageLevel")
+                .equipmentName("NewVoltageLevel")
+                .nominalVoltage(379.3)
+                .substationId("s1")
+                .nominalVoltage(379.1)
```

Comment:
useless ... nominalVoltage is set just above ...

Reply:
Done